cluster/gce/config-default.sh

```diff
@@ -60,7 +60,7 @@ fi
 # variable. Also please update corresponding image for node e2e at:
 # https://github.com/kubernetes/kubernetes/blob/master/test/e2e_node/jenkins/image-config.yaml
 CVM_VERSION=${CVM_VERSION:-container-vm-v20170214}
-GCI_VERSION=${KUBE_GCI_VERSION:-gci-stable-56-9000-84-2}
+GCI_VERSION=${KUBE_GCI_VERSION:-cos-stable-57-9202-64-0}
```

Comment:
Probably more location too - see https://github.com/kubernetes/kubernetes/pull/41819/files as a previous example of bumping up COS image version.